src/renderer/services/deposit/ethers.ts

```diff
@@ -0,0 +1,82 @@
+import {ethers, utils, Contract} from "ethers";
+import {DEPOSIT_AMOUNT} from "./constants";
+import {bool, Gwei} from "@chainsafe/eth2.0-types";
+import DepositContract from "./options";
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+import {deserialize} from "@chainsafe/ssz";
+import BN from "bn.js";
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+
+const PUBKEY_INDEX = 0;
+const DATA_INDEX = 2;
+const DEPOSIT_AMOUNT_GWEI = new BN(DEPOSIT_AMOUNT).imul(new BN(ethers.utils.parseUnits("1", "gwei").toString()));
+const DEPOSIT_EVENT = "DepositEvent";
+
+export class EthersNotifier {
+    private provider: ethers.providers.Web3Provider;
+    private depositContractAddress: string;
+    private config: IBeaconConfig;
+    private signingKey: Keypair;
+
+    public constructor(
+        provider: ethers.providers.Web3Provider,
+        depositContractAddress: string,
+        config: IBeaconConfig,
```

Comment:
I was thinking, since we will support various testnets,
you could create interface:
```javascript
export interface NetworkConfig {
   config: IBeaconConfig;
   contract: {
      address: string,
      bytecode: string,
      deployedAtBlock: number
   }
}
```